act-rules-format.md

```diff
@@ -254,6 +256,28 @@ If the rule does not map to any [=accessibility requirement=], the accessibility
 If the `failed` [=outcome=] cannot be mapped to an [=accessibility requirement=], there <em class="rfc2119">must not</em> be an accessibility requirement in the accessibility requirements mapping. The optional [Background section](#background) could be used to list [=accessibility requirements documents=] when they are thematically related, but for which the rule is not a failure test.
 
 
+### Implementing Accessibility Requirement Mappings ### {#implementing-accessibility-requirement-mappings}
+
+[=Implementations=] can only have a [=complete=] [=consistency=] level if they reports accessibility requirements as *not satisfied* in a way that is consistent with the accessibility requirements mapping. A set of [=implementation procedures=] has a <dfn>consistent accessibility requirements mapping</dfn> when all `failed` test cases of an ACT Rule are reported as *not satisfied* for each [=relevant accessibility requirement=] by at least one implementation procedure in the set. Additionally, no other accessibility requirement in the same [=accessibility requirements documents=] is reported as *not satisfied* by the implementation procedures in the set.
```

Comment:
First comment! Sure I will screw it up.  I have read line 261 several times and I am having a hard time understanding what it means. This sentence in particular confounds me: "[=Implementations=] can only have a [=complete=] [=consistency=] level if they report accessibility requirements as *not satisfied* in a way that is consistent with the accessibility requirements mapping. "